src/lib/components/BlogLink.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+  import { formatDate } from "$lib/utils/services/datefmt";
+  import type { EndPoints } from "$lib/utils/types/microcms";
+
+  export let blogData: EndPoints["get"]["blogs"];
+  const { id, title, publishedAt, tags, ogpImg } = blogData;
+</script>
+
+<div>
+  <div class="ogp">
+    <a href={`/blogs/${id}`}>
+      {#if ogpImg}
+        <img
+          src={ogpImg.url}
+          alt={title}
+          height={ogpImg.height}
+          width={ogpImg.width}
+        />
+      {:else}
+        <img src="https://placehold.jp/1200x630.png" alt="ogp" />
+      {/if}
+    </a>
+  </div>
+  <p>公開：{formatDate(publishedAt)}</p>
+  <ul>
+    {#each tags as tag (tag)}
+      <li>{tag}</li>
+    {/each}
+  </ul>
+  <a href={`/blogs/${id}`}><h3>{title}</h3></a>
```

Comment:
基本的にインライン要素の aタグ の中にブロック要素の hタグ を入れるのは良くないから
```suggestion
  <h3><a href={`/blogs/${id}`}>{title}</a></h3>
```
とした方が良い